pkgs/development/interpreters/clojure/lumo.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, fetchFromGitHub, boot, nodejs-8_x, yarn, python }:
+
+stdenv.mkDerivation rec {
+  name = "lumo-${version}";
+  version = "1.8.0";
+
+  src = fetchFromGitHub {
+    owner = "anmonteiro";
+    repo = "lumo";
+    rev = "fb97ce5086928252b8dd5b862a2d62d5e3daf825";
+    sha256 = "0ci9rgpaww558sbvxkpf69rlkc4029ih70rja9vlr9a2nsp72cqp";
+  };
+
+  nativeBuildInputs = [
+    boot
+    nodejs-8_x
+    python
+    yarn
+  ];
+
+  buildPhase = ''
+     # remove audible build notifications
+     sed -i '/notify :audible true/d' build.boot
+
+     mkdir boot_home m2_home
+
+     BOOT_HOME=boot_home BOOT_LOCAL_REPO=m2_home \
+       boot release
```

Comment:
This requires network access which is not allowed in nix's sandbox build:

```
building '/nix/store/ll0w200d8lm0y0z3rn8hr27kbq43mh8d-lumo-1.8.0.drv'...
unpacking sources
unpacking source archive /nix/store/j5wh7z9jnzjx0dfa8g6ivsbyj6kfs276-source
source root is source
patching sources
configuring
no configure script, doing nothing
building
Downloading https://github.com/boot-clj/boot/releases/download/2.7.2/boot.jar...
Exception in thread "main" java.net.UnknownHostException: github.com
        at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:184)
        at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
        at java.net.Socket.connect(Socket.java:589)
        at sun.security.ssl.SSLSocketImpl.connect(SSLSocketImpl.java:673)
        at sun.security.ssl.BaseSSLSocketImpl.connect(BaseSSLSocketImpl.java:173)
        at sun.net.NetworkClient.doConnect(NetworkClient.java:180)
        at sun.net.www.http.HttpClient.openServer(HttpClient.java:463)
        at sun.net.www.http.HttpClient.openServer(HttpClient.java:558)
        at sun.net.www.protocol.https.HttpsClient.<init>(HttpsClient.java:264)
        at sun.net.www.protocol.https.HttpsClient.New(HttpsClient.java:367)
        at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.getNewHttpClient(AbstractDelegateHttpsURLConnection.java:191)
        at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1156)
        at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:1050)
        at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:177)
        at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1564)
        at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1492)
        at sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:263)
        at java.net.URL.openStream(URL.java:1045)
        at Boot.download(Boot.java:176)
        at Boot.install(Boot.java:227)
        at Boot.main(Boot.java:241)
builder for '/nix/store/ll0w200d8lm0y0z3rn8hr27kbq43mh8d-lumo-1.8.0.drv' failed with exit code 1
error: build of '/nix/store/ll0w200d8lm0y0z3rn8hr27kbq43mh8d-lumo-1.8.0.drv' failed
```

Maybe it is possible to create a fixed-input derivation, that creates all the source code.
But I am not sure, how this would work for closure script.